exec/java-exec/src/main/java/org/apache/drill/exec/store/ischema/Records.java

```diff
@@ -543,4 +550,38 @@ public Schema(String catalog, String name, String owner, String type, boolean is
       this.IS_MUTABLE = isMutable ? "YES" : "NO";
     }
   }
-}
+
+  public static class File {
```

Comment:
Could you please add Javadoc similar to Javadocs for other nested classes in this class.